src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -73,7 +72,7 @@
      */
     private static String getUser(Request req, Response res) {
         HttpGet request = new HttpGet(getUserIdUrl(req));
-        setHeaders(request);
+        setHeaders(req, request);
```

Comment:
Can we rename `request` to `getUserRequest` to avoid having two very similarly named variables?